great_expectations/dataset/sqlalchemy_dataset.py

```diff
@@ -1222,6 +1222,10 @@ def create_temporary_table(self, table_name, custom_sql, schema_name=None):
             stmt = 'CREATE TEMPORARY TABLE "{table_name}" AS {custom_sql}'.format(
                 table_name=table_name, custom_sql=custom_sql
             )
+
+        # close_with_result = True
```

Comment:
You might want to remove these commented out lines

Reply:
Great catch - thanks!